execution/executor/src/lib.rs

```diff
@@ -678,9 +681,22 @@ where
             next_validator_set = vm_output
                 .events()
                 .iter()
-                .find(|event| *event.key() == validator_set_change_event_key)
-                .map(|event| ValidatorSet::from_bytes(event.event_data()))
-                .transpose()?
+                .find(|event| {
+                    *event.key() == validator_set_change_event_key
+                        || *event.key() == new_epoch_event_key
```

Comment:
isn't new_epoch_event_key sufficient? isn't one event a superset of the other?

Reply:
no, next commit will remove validator_change_event. currently it's not part of the libra config yet